install/Knative-custom-install.md

```diff
@@ -60,26 +57,28 @@ If you are just getting started with Knative, you should choose automatic
 sidecar injection and enable the Istio service mesh.
```

Comment:
Revise this to include the "standard install suitable for most users" (re: comment below).  Maybe:

"**Tip**: If you are just getting started with Knative or want to ensure that your installation is compatible with most use cases, you should install `istio-crds.yaml` and `istio.yaml` to enable the Istio service mesh through automatic sidecar injection."